.github/workflows/deploy-to-aks.yaml

```
@@ -38,7 +38,7 @@ jobs:
       - name: Install CLI tools
         run: |
           pip install tox
-          sudo snap install juju --channel=${{ env.JUJU_VERSION }}/stable
+          sudo snap install juju --channel=${{ env.JUJU_VERSION }}/latest/stable
```

issue: I think we will encounter an issue here, since the channel `3.6/latest/stable` doesn't exist. I think you probably meant to keep `3.6/stable`.
```suggestion
          sudo snap install juju --channel=${{ env.JUJU_VERSION }}/latest/stable
```